lib/pf/condition/network.pm

```diff
@@ -44,11 +40,17 @@ match the last ip to see if it is in defined network
 =cut
 
 sub match {
-    my ($self,$data) = @_;
-    my $last_ip = $data->{last_ip} if exists $data->{last_ip};
-    return defined $last_ip && $self->value->contains(NetAddr::IP->new($last_ip));
+    my ($self,$ip) = @_;
+    return 0 unless defined $ip;
```

Comment:
use the $FALSE james

![image](https://cloud.githubusercontent.com/assets/3857942/9940477/2046d302-5d3e-11e5-84c3-7be4a412f527.png)